src/main/controllers/start_date/StartDateController.ts

```diff
@@ -0,0 +1,38 @@
+import { Response } from 'express';
+
+import { Form } from '../../components/form/form';
+import { AppRequest } from '../../definitions/appRequest';
+import { StillWorking } from '../../definitions/case';
+import { PageUrls, TranslationKeys } from '../../definitions/constants';
+import { FormContent, FormFields } from '../../definitions/form';
+import { assignFormData, getPageContent, handleSessionErrors, setUserCase } from '../helpers';
+
+export default class startDateController {
+  private readonly form: Form;
+
+  constructor(private readonly startDateContent: FormContent) {
+    this.form = new Form(<FormFields>this.startDateContent.fields);
+  }
+  public post = (req: AppRequest, res: Response): void => {
+    setUserCase(req, this.form);
+    let redirectUrl = '';
+    if (req.session.userCase.isStillWorking === StillWorking.WORKING) {
+      redirectUrl = PageUrls.NOTICE_PERIOD;
+    } else if (req.session.userCase.isStillWorking === StillWorking.NOTICE) {
+      redirectUrl = PageUrls.NOTICE_END;
+    } else if (req.session.userCase.isStillWorking === StillWorking.NO_LONGER_WORKING) {
+      redirectUrl = PageUrls.HOME;
```

Comment:
Please put a TODO in here so it's clear this needs to be wired up

Reply:
Added TODO